src/AccessibilityInsights.SharedUx/Dialogs/HotkeyGrabDialog.xaml.cs

```diff
@@ -117,18 +102,23 @@ private string GetCleanKeyString()
             return str;
         }
 
-        /// <summary>
-        /// Handles key up. Closes window on esc.
-        /// </summary>
-        /// <param name="sender"></param>
-        /// <param name="e"></param>
         private void Window_KeyUp(object sender, KeyEventArgs e)
         {
             if (e.Key == Key.Escape)
             {
                 e.Handled = true;
                 this.Close();
-            } 
+            }
+        }
+
+        private void TbHotkey_PreviewTextInput(object sender, TextCompositionEventArgs e) => e.Handled = true;
+
+        private void TbHotkey_PreviewExecuted(object sender, ExecutedRoutedEventArgs e)
+        {
+            if (e.Command == ApplicationCommands.Paste)
```

Comment:
Should this also cover other types of ApplicationCommands that would normally modify the element's Text? (looking at the list of them, Cut and Delete come to mind)